src/app/shared/example-viewer/example-viewer.html

```diff
@@ -1,37 +1,61 @@
 <div class="docs-example-viewer-wrapper">
-  <div class="docs-example-viewer-title">
+  <div class="docs-example-viewer-title" *ngIf="view !== 'compact'">
     <div class="docs-example-viewer-title-spacer">{{exampleData?.title}}</div>
 
-    <button mat-icon-button type="button" (click)="toggleSourceView()"
-            [matTooltip]="'View ' + exampleData?.title + ' source'" aria-label="View source">
+    <button mat-icon-button type="button" (click)="toggleCompactView()" [matTooltip]="'Show snippet only'"
+            aria-label="View less" *ngIf="showCompactToggle">
       <mat-icon>
-        <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fit="" preserveAspectRatio="xMidYMid meet" focusable="false">
-          <path fill="none" d="M0 0h24v24H0V0z"></path>
-          <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"></path>
-        </svg>
+      <svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
I see the original svg had some attributes set, most of them probably aren't needed if it looks right without them, but `focusable="false"` looks like it should at least stay

Reply:
Actually, just brought up a demo and if I inspect the <svg> element, it's huge, so we may need the `width="100%"` and `height="100%"` too